client/src/pages/Details.js

```diff
@@ -70,6 +137,15 @@ export default function Details({ member, updateMember, members }) {
         </DetailsHeadline>
         <DetailsGroup>{newMember.group}</DetailsGroup>
         <p>{newMember.description}</p>
+        <SavedNoteWrapper>
+          {savedNotes.length >= 1 &&
```

Comment:
`savedNotes` will always be an array so it is save to map over it without the guard